### PR TITLE
Support iottalk v1 dan api

### PR DESCRIPTION
**Pull request by Ksoy**

### CODE REVIEW

File: src/dan2.js [hunk 1]
```diff
@@ -103,8 +101,8 @@ const register = function(url, params, callback) {
         .set('Accept', '*/*')
         .send(JSON.stringify({
             'name': params['name'],
-            'idf_list': params['idf_list'],
-            'odf_list': params['odf_list'],
+            'idf_list': params['df_list'] ? params['df_list'] : params['idf_list'],
```
Comment (iblislin) [inline src/dan2.js]: what aboue `(params.df_list === undefined ?) ... : ...` ?

Comment (Ksoy) [reply src/dan2.js]: ok, sounds good

File: src/dan2.js [hunk 2]
```diff
@@ -147,6 +147,7 @@ const register = function(url, params, callback) {
                     'payload': JSON.stringify({'state': 'broken', 'rev': _rev}),
                     'retain': true,
                 },
+                'keepalive': 60
```
Comment (iblislin) [inline src/dan2.js]: I'm ok with lower value

**Comment by Ksoy** [comment]
bad

**Pull request closed without merging**